src/core/common/ResourseTemplates/CreateTemplate.js

```diff
@@ -44,14 +60,24 @@ const styles = theme => ({
  * (it used in Create blocks for the plugins Allergies, Contacts, Medications, Problems etc.)
  *
  * @author Bogdan Shcherban <bsc@piogroup.net>
- * @param {shape}  classes
- * @param {string} blockTitle
- * @param {shape}  children
- * @param {shape}  rest
+ * @param {shape}   classes
+ * @param {boolean} isListOpened
+ * @param {func}    toggleListBlock
+ * @param {string}  blockTitle
+ * @param {shape}   children
+ * @param {shape}   rest
  */
-const CreateTemplate = ({ classes, blockTitle, children, ...rest }) => (
-    <Grid item xs={12} sm={6} className={classes.createBlock}>
-        <Typography className={classes.blockTitle}>{blockTitle}</Typography>
+const CreateTemplate = ({ classes, isListOpened, toggleListBlock, blockTitle, children, ...rest }) => (
+    <Grid item xs={12} sm={isListOpened ? 6 : 12} className={classes.createBlock}>
+        <div className={classes.blockTitle}>
+            <Typography className={classes.title}>{blockTitle}</Typography>
+            <div className={classes.emptyBlock}></div>
+            <Tooltip title={isListOpened ? "Expand" : "Compress"}>
+                <IconButton onClick={() => toggleListBlock()}>
```

Comment:
As explained before, inlining functions as props has performance impacts as it forces rerendering. Please read https://medium.com/dailyjs/react-is-slow-react-is-fast-optimizing-react-apps-in-practice-394176a11fba for details.